.github/workflows/helm-release.yml

```diff
@@ -0,0 +1,52 @@
+name: helm releaser
+
+on:
+  push:
+    branches: [master, main, force_test, release-*, staging, trying]
+    tags: ['*']
+  pull_request:
+    branches: ['**']
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.7.1
+
+      - name: Download latest index.yml
+        run: |
+          mkdir -p chart_release/
+          wget https://charts.timescale.com/index.yaml -O chart_release/index_old.yaml
+
+      - name: Create package
+        run: |
+          mkdir -p chart_release/packaged
+          helm package deploy/helm-chart -d chart_release/packaged
+          helm repo index chart_release/packaged --merge=chart_release/index_old.yaml  --url http://charts.timescale.com
+          # debug to be deleted
+          ls chart_release/packaged
+          cat chart_release/packaged/index.yaml
+
+#now send to s3
+#      - name: upload to s3
+#        if: startsWith(github.ref, 'refs/tags/')
+#        uses: jakejarvis/s3-sync-action@master
+#        with:
+#          args: --acl public-read
+#        env:
+#          SOURCE_DIR: chart_release/packaged/
+#          AWS_S3_BUCKET: ${{ secrets.AWS_S3_BUCKET }}
+#          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+#          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+#          AWS_REGION: 'us-east-1'
+#
+#
```

Comment:
Can we put relevant parts in `.github/workflows/helm.yaml` so we don't need to track multiple files?